### PR TITLE
Fix/skale 2658 fix commission rate units

### PR DESCRIPTION
**Pull request by dmytrotkk**

### CODE REVIEW

File: utils/print_formatters.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def print_validators(validators, wei):
         'Id',
         'Address',
         'Description',
-        'Fee rate (permille)',
+        'Fee rate (precent %)',
```
Comment (cstrangedk) [inline utils/print_formatters.py]: typo -> percent

File: utils/print_formatters.py [hunk 2]
```diff
@@ -67,12 +67,13 @@ def print_validators(validators, wei):
...
             validator['minimum_delegation_amount'] = from_wei(
                 validator['minimum_delegation_amount'])
+        fee_rate_percents = permille_to_percent(validator['fee_rate'])
         rows.append([
             validator['name'],
             validator['id'],
             validator['validator_address'],
             validator['description'],
-            validator['fee_rate'],
+            fee_rate_percents,
```
Comment (cstrangedk) [inline utils/print_formatters.py]: fee_rate_percent

File: text.yml [hunk 3]
```diff
@@ -25,8 +25,8 @@ validator:
       help: Validator description
       prompt: Please enter validator description
     commission_rate:
-      help: Commission rate (permille ‰)
-      prompt: Please enter validator commission rate in permille (0 - 1000)
+      help: Commission rate (percents %)
```
Comment (cstrangedk) [inline text.yml]: percent

File: text.yml [hunk 4]
```diff
@@ -25,8 +25,8 @@ validator:
...
+      prompt: Please enter validator commission rate in percents (0 - 100)
```
Comment (cstrangedk) [inline text.yml]: percent

**Review by cstrangedk — CHANGES_REQUESTED** [review]
lgtm, just some typos

**Comment by dmytrotkk** [comment]
@cstrangedk typos fixed

**Pull request merged**